docs/components/dropdown/examples/FwbDropdownExampleAlignment.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="vp-raw flex gap-2 flex-wrap">
+    <fwb-dropdown
+      placement="top"
+      text="Top"
+      align-to-end
+    >
+      <p class="p-2">
+        Dropdown content here
+      </p>
```

Comment:
Can we please change dropdown contents to something like (in every example):
```html
        <div class="w-52">
          <p class="p-2">Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
        </div>
```

This will make this change way more visible.